src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Association class (person and tag) that represents an edit-tag operation that will be printed after a session
+ *
+ */
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private String edit;
+    
+    public Tagging(Person person, Tag tag, String edit) {
```

Comment:
avoid using variable/parameter name such as `edit`. it's very vague and other developers have to guess what it means.